### DISCUSSION TITLE
How to change Security Onion web interface?

### DISCUSSION THREAD

**Discussion by w0lfcat** [Security-Onion-Solutions/securityonion#2397] [Unsupported Versions] [answered]
I've just installed Security Onion 2.3.10 in VirtualBox and noticed that the web interface was configured on wrong interface. I've 3 network interfaces in this box (host-only, internal, nat) and would like to change the management interface.

```
Access the Security Onion web interface at https://x.x.x.x
(You may need to run so-allow first if you haven't yet) 
```

What is the right way to do this?

I've no idea why SO decided to use this IP when it's not reachable at all. I can't even ping it from SO box.

**Update**
I've remove NAT interface from VirtualBox and did not get that message anymore. However, I still can't access to web UI of this box. Port 80/443 is not listening at all.

**Comment by weslambert** [comment, marked as answer]
I'd perform a fresh install if possible, ensuring that the correct interface is chosen, and that you have internet connectivity.

**Comment by AndriiStepovyi** [comment]
ok, first simple and short answer:
>sudo so-ip-update
#changes SO Manager web access IP

aside from that you may change actual interface IP by:
>sudo ntmui
#CentOS network setup interface

In my case I have changed interface first, but seemed like old IP was still written and prevented me from web access via New One.
So check load your system, record which web access IP it is pointing currently, compare with appointed Network Interface and if different- use so-ip-update to rewrite the web access ip. Just do not forget to 
>sudo so-allow
#to pinhole firewall for that ip

**Comment by chmelarp** [comment]
There is a problem in the redirect it does for no reason...
I have a VirtualBox NAT, which basically means the internal IP is 10.0.2.15. but when I make port forwarding (to localhost), it always redirects to 10.0.2.15 on HTTPS/443 (why the heck), but never on HTTP/80 - you get just Welcome to nginx!
Funny!

**Reply by abooosin** [reply]
Did you find any solution?

**Reply by chmelarp** [reply]
I did it somehow, but don't remember how as it is almost one year and I scraped the image a long time ago.
